src/shared/ka-error.js

```diff
@@ -12,17 +12,25 @@ export default class KAError<TKind: ErrorKind> extends Error {
     sourceError: ?SimplifiedError;
 
     /**
-     * Construct KAError instance.
+     * Construct a KAError instance.
      *
      * @param {string} message The error message.
-     * @param {TKind} kind The kind of KA error.
+     * @param {TKind} kind The kind of error.
      * @param {Error} [sourceError] The original error that spawned this one.
      */
     constructor(message: string, kind: TKind, sourceError: ?Error = null) {
         super(message);
+
+        // Set the name so we get a nice error output, like
+        // KAInternalError
```

Comment:
These comments should force the build to generate new source map which will then cause us to push a new dist.